### PR TITLE
Add NWB export pytests

### PR DESCRIPTION
**Pull request by kabilar**
- Works within Docker environment and starting with empty schemas.
- Depends on [datajoint/element-array-ephys #44](https://github.com/datajoint/element-array-ephys/pull/44) and [datajoint/element-array-ephys #58](https://github.com/datajoint/element-array-ephys/pull/58).

### CODE REVIEW

**Comment by kabilar** [comment]
Hi @bendichter, I copied over your pytests from Slack and made some minor modifications. Please take a look and let me know if you see any issues. Thanks!

File: workflow_array_ephys/pipeline.py [hunk 1]
```diff
@@ -36,8 +38,15 @@ class SkullReference(dj.Lookup):
...
-# Activate "ephys" schema -----------------------------------------------------
+# Activate "ephys" schema --------------------------------------------------------------
 
 ephys.activate(db_prefix + 'ephys', 
                db_prefix + 'probe', 
                linking_module=__name__)
+
+# Import NWB export functions ----------------------------------------------------------
+
+from element_lab.export.nwb import element_lab_to_nwb_dict
```
Comment (CBroz1) [inline workflow_array_ephys/pipeline.py:49]: I started to draft this change on my fork, but my linter complained: `E402  module level import not at top of file`

Importing the export function needs to occur after the respective element has been activated, so the solve I landed on was declaring `workflow_array_ephys/export.py` with the relevant imports there. What do you think @kabilar? No need to include in this PR

File: tests/test_export.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+     session_key = dict(subject='subject5', session_datetime='2018-07-03 20:32:28')
+     
+     ephys.LFP.populate(session_key, display_progress=True)
+     ephys.CuratedClustering.populate(session_key, display_progress=True)
+     ephys.WaveformSet.populate(session_key, display_progress=True)
+     
+     nwbfile=ecephys_session_to_nwb(session_key=session_key,
+          raw=False,
+          spikes=False,
```
[the remark this answers is not in the document]
Comment (kabilar) [reply tests/test_export.py:56]: Thanks @CBroz1.  Good point.  I will update.

**Comment by bendichter** [comment]
The tests look fine to me

**Comment by kabilar** [comment]
Update to README:
- Add citation section.
- Update links to point to elements.datajoint.org instead of the datajoint-elements GitHub repository.
- Remove images of upstream Elements to shorten README.

**Pull request merged**